GetGitRevisionDescription.cmake

```diff
@@ -84,38 +84,21 @@ function(git_describe _var)
 	if(NOT GIT_FOUND)
 		find_package(Git QUIET)
 	endif()
-	get_git_head_revision(refspec hash)
```

Comment:
I think the reason I originally did this was to ensure that a git commit would trigger a re-CMake if desired. Does this behavior still work with your changes?